specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -608,6 +751,94 @@
           "type": "string"
         }
       }
+    },
+    "PutAliasRequest": {
+      "description": "The parameters required to create a new subscription.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PutAliasRequestProperties",
+          "description": "Put alias request properties."
+        }
+      }
+    },
+    "PutAliasRequestProperties": {
+      "description": "Put subscription properties.",
+      "required": [
+        "displayName",
+        "workloadType",
+        "billingScope"
+      ],
+      "properties": {
+        "displayName": {
+          "description": "The friendly name of the subscription.",
+          "type": "string"
+        },
+        "workloadType": {
+          "description": "The workload type of the subscription. It can be either Production or DevTest.",
```

Comment:
Make this a string enum with the allowed values please.  https://github.com/stankovski/AutoRest/blob/master/Documentation/swagger-extensions.md#x-ms-enum